apisix/plugins/fault-injection.lua

```diff
@@ -28,26 +29,28 @@ local schema = {
                 http_status = {type = "integer", minimum = 200},
                 body = {type = "string", minLength = 0},
```

Comment:
Should we set a default value for http_status and body property?

Reply:
we do not need default value. the default `http_status` is `200` for nginx.